src/luascript.cpp

```diff
@@ -10280,7 +10282,7 @@ int LuaScriptInterface::luaPlayerSetEditHouse(lua_State* L)
 
 int LuaScriptInterface::luaPlayerSetGhostMode(lua_State* L)
 {
-	// player:setGhostMode(enabled[, showEffect=true])
+	// player:setGhostMode(enabled[, magicEffect=CONST_ME_TELEPORT])
```

Comment:
shouldnt this be `magicEffect = CONST_ME_TELEPORT`?

Reply:
Sure, adjusted it now.